src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
-            page.StandardOperators.NumberPad.Input(3.3);
-            page.MemoryPanel.MemMinus.Click();
-            var memoryItems = page.MemoryPanel.GetAllMemoryListViewItems();
-            Assert.IsTrue(memoryItems[0].Text.Equals("0.7", StringComparison.InvariantCultureIgnoreCase));
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            page.StandardAoTCalculatorPage.AoTWindowSizeWithinRange();
+            Size smallAoTWindowSize = new Size(161, 168);
+            Size largeAoTWindowSize = new Size(502, 502);
+            Size mediumAoTWindowSize = new Size(396, 322);
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallAoTWindowSize;
+            Assert.AreEqual("(161, 168)", page.CalculatorApp.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeAoTWindowSize;
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = mediumAoTWindowSize;
+            Assert.AreEqual("(396, 322)", page.CalculatorApp.GetCalculatorWindowSize());
+            page.StandardAoTCalculatorPage.NavigateToStandardMode();
         }
 
         [TestMethod]
-        public void Memory_RecallTest()
+        [Priority(2)]
+        public void AoT_ScaleRetention()
         {
-            page.StandardOperators.NumberPad.Input(12);
-            page.StandardOperators.DivideButton.Click();
-            page.StandardOperators.NumberPad.Input(3);
-            page.StandardOperators.EqualButton.Click();
+            Size smallWindowSize = new Size(464, 502);
+            Size largeAoTWindowSize = new Size(502, 502);
+
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallWindowSize;
+            Assert.AreEqual("(464, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeAoTWindowSize;
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardMode();
+            Assert.AreEqual("(464, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
 
-            page.MemoryPanel.MemButton.Click();
-            page.StandardOperators.NumberPad.Input(3.3);
-            page.MemoryPanel.MemRecall.Click();
-            var memoryItems = page.MemoryPanel.GetAllMemoryListViewItems();
-            Assert.IsTrue(memoryItems[0].Text.Equals("4", StringComparison.InvariantCultureIgnoreCase));
         }
 
         [TestMethod]
-        public void Memory_ClearTest()
+        [Priority(2)]
+        public void AoT_ErrorMessage()
```

Comment:
This should be broken into 3 different tests

Reply:
I broke this up into two tests:
- AoT_ErrorMessage_ResultUndefined
- AoT_ErrorMessage_CannotDivideByZero
- AoT_ErrorMessage_ MessageRetentionUponExitingAoT

Change will be in next commit